packages/components/src/theme/theme-dark/theme-dark-semantic.css

```diff
@@ -199,4 +199,7 @@
 
   /* Editing */
   --dh-color-modified: var(--dh-color-yellow-1300);
+
+  /* Components */
+  --dh-toggle-switch-bg: var(--dh-color-gray-500);
```

Comment:
We actually have `theme-xxx-components.css` for component specific variables. Alternatively, could also just define it as a local var inline
```typescript
span[class*='spectrum-ToggleSwitch-switch'] {
  --dh-toggle-switch-bg: var(--dh-color-gray-500);
  
  /* increase contrast of switch in off position */
  background: var(--dh-toggle-switch-bg);
}
```

Reply:
yeah should go in the components file, both light and dark. And then check that it looks good in light mode still because I didn't and adjust it if necessary.